apps/vx-scan/backend/src/app.ts

```diff
@@ -237,6 +241,17 @@ function buildApi(
       return result.isErr() ? result : ok();
     },
 
+    async getCastVoteRecordsForTally(): Promise<CastVoteRecord[]> {
```

Comment:
I suggest we mark this as `@deprecated` so we don't expand its usage.

```suggestion
    /**
     * @deprecated we should build the tally in the backend instead
     */
    async getCastVoteRecordsForTally(): Promise<CastVoteRecord[]> {
```

<img width="586" alt="image" src="https://user-images.githubusercontent.com/1938/210842502-1a7ef96d-147d-4075-890d-99a2bfeb3f9f.png">

Reply:
Will do!